### DISCUSSION TITLE
Postgres persistance default schema

### DISCUSSION THREAD

**Discussion by shekarpcs** [Netflix/conductor#2908] [Q&A]
All, When I configure postgres persistance , it is using public schema by default. public schema is disabled in our environment. 

How to set the another schema for postgres SQL?

it is high priority.

**Comment by apanicker-nflx** [comment]
@rickfish @mactaggart Could you please help with this? Thanks

**Comment by rickfish** [comment]
You need to add &currentSchema=<schema> to your postgres url

**Reply by erikbrgr** [reply]
This won’t work as the line `.schemas(“public”)` in `PostgresConfiguration.java` will override it. I wrote an article [here](https://betterprogramming.pub/running-netflix-conductor-3-in-docker-using-elasticsearch-7-and-postgresql-b415988dd74a) that addresses this, and commented on a PR review. Unfortunately, I can’t find it anymore :blush:

**Comment by shekarpcs** [comment]
The code is defaulting to public schema which is vonurable and many orga
disable this.

This is making conductor unusable in production with postgress

**Reply by erikbrgr** [reply]
In my article [Running Conductor 3 …](https://betterprogramming.pub/running-netflix-conductor-3-in-docker-using-elasticsearch-7-and-postgresql-b415988dd74a) under Postgres, I explain how to fix this. You need to remove the `.schemas(“public”)` line from `PostgresConfiguration.Java`

**Comment by shekarpcs** [comment]
We do not want to get into changing code. We are getting conductor jar from
mavin repo. We are waiting for person who added this to remove in upcoming
release . Or we are stuck at 3.4.1 as 4..2 updated with this public schema
thingy

**Reply by jxu-nflx** [reply]
@shekarpcs Would you be interested to create a PR to fix it by following the steps in @erikbrgr 's [article](https://betterprogramming.pub/running-netflix-conductor-3-in-docker-using-elasticsearch-7-and-postgresql-b415988dd74a)?
